app/models/User.ts

```diff
@@ -46,6 +46,12 @@ export enum CfpVote {
   NEUTRAL = "Neutral",
 }
 
+export enum AmlCheck {
+  PASS = "Pass",
+  FAIL = "Fail",
+  PENDING = "Pending",
+}
```

Comment:
Because we have other information like KYC in there too and therefore thought maybe it would make sense to also put AML enum here. But I have no problem to move it somewhere else :)

Reply:
AML is only used for transactions, so the RouteHistory.ts might be a better place 🤔